.gitignore

```diff
@@ -55,3 +55,5 @@ lib*.pc
 # style related temporary outputs
 *.uncrustify
 .venv
+
+citus-tools
```

Comment:
As "tools" is a separate repo, I don't see a big reason to add this into .gitignore. Any reasons for doing that?

Reply:
I think we can use our local "global gitignores" for such purposes.